app/src/main/java/com/infomaniak/drive/ui/MainViewModel.kt

```diff
@@ -21,15 +21,7 @@ import android.app.Application
 import android.content.Context
 import android.provider.MediaStore
 import androidx.fragment.app.FragmentActivity
-import androidx.lifecycle.AndroidViewModel
-import androidx.lifecycle.LiveData
-import androidx.lifecycle.MediatorLiveData
-import androidx.lifecycle.MutableLiveData
-import androidx.lifecycle.SavedStateHandle
-import androidx.lifecycle.asLiveData
-import androidx.lifecycle.liveData
-import androidx.lifecycle.switchMap
-import androidx.lifecycle.viewModelScope
+import androidx.lifecycle.*
```

Comment:
We need to check your AS settings to avoid this
We don't want wildcard imports anymore (in order to follow android's guideline)